marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
So, HttpClientManager is the class that provides pool of HttpClientConnections and contains convenient facade to HttpClientBuilder that needed to pentaho products. HttpClientUtil - auxiliary class in order to not duplicate logic in a lot of places. It easily can be expanded. I'm not sure that this logic is enough for new module creation. I can add duplicate this in marketplace and that's it for now. Any objections?